bcipy/parameters/parameters.json

```diff
@@ -876,5 +876,13 @@
     "helpTip": "What should the name of the main log file be for each BciPy session",
     "recommended_values": "",
     "type": "str"
+  }, 
+  "section_names": {
+    "value": "{'top_level_config': 'Important BciPy Parameters', 'acq_config': 'ACQ Configuration', 'bci_config': 'BCI Configuration', 'feedback_config': 'Feedback Configuration', 'lang_model_config': 'Language Model Configuration', 'signal_config': 'Signal Configuration'}",
+    "section": "bci_config",
+    "readableName": "Names of parameter sections",
+    "helpTip": "Names of parameter sections",
+    "recommended_values": "",
+    "type": "object"
```

Comment:
The 'section_names' here is a developer-focused configuration rather than a user-focused parameter. My feeling is that this should go elsewhere, either in a config file or somewhere in the GUI code. 

As for the section names themselves: ACQ is not an abbreviation; it should just be spelled out as Acquisition. All of this is configuration so having 'Configuration' in the section is redundant. The BCI section is really large and we should brainstorm ways of breaking it down; also, maybe this should be named General.